src/doc/guide.md

```diff
@@ -383,30 +388,32 @@ fn main() {
 }
 ```
 
-Putting `fn main() {` in each example is a bit tedious, so we'll leave that out
-in the future. If you're following along, make sure to edit your `main()`
-function, rather than leaving it off. Otherwise, you'll get an error.
+**Note**: Wrapping `fn main() { }` around each example is tedious, so we'll act
```

Comment:
yeah, I think I had it in blockquote before, and it looked better. I can remove and rephrase to make it part of the sentence.